app/views/editorial/nodes/options/_toc.html.haml

```diff
@@ -0,0 +1,3 @@
+-# locals: f
+= f.input :toc, collection: 0..2, label: 'Table of contents',
```

Comment:
Yes, they do feel like magic and yes, they do refer to depth. But again, I think it will be easier to fix this up once we have a second option and the abstraction is easier to figure out.